lib/fixtures/LICENSE.md

```diff
@@ -1,6 +1,6 @@
 #Learn.co Educational Content License
 
-Copyright (c) 2015 Flatiron School, Inc
```

Comment:
does the copyright refer to when the labs are created, or when the license was created?